transport/src/commonMain/kotlin/com/genesys/cloud/messenger/transport/core/IAttachmentHandler.kt

```diff
@@ -0,0 +1,36 @@
+package com.genesys.cloud.messenger.transport.core
+
+import com.genesys.cloud.messenger.transport.shyrka.receive.PresignedUrlResponse
+import com.genesys.cloud.messenger.transport.shyrka.receive.UploadSuccessEvent
+import com.genesys.cloud.messenger.transport.shyrka.send.DeleteAttachmentRequest
+import com.genesys.cloud.messenger.transport.shyrka.send.OnAttachmentRequest
+
+internal interface IAttachmentHandler {
```

Comment:
I don't think we've used this pattern of prefixing interfaces with `I`. Is it OK to simply call this `AttachmentHandler`?

Reply:
@crumpf  good point. 
It is more than OK. That was my initial approach, but then I realize that the `diff` this naming produce makes it difficult to see the changes I made on the implementation class. I guess this is not a good enough reason. I will follow your suggestion and rename to AttachmentHandler/AttachmentHandlerImpl .